dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/HdfsPermissionExtractionConnector.java

```diff
@@ -46,10 +47,11 @@
     description = "The size of the thread pool to use when extracting hdfs permissions.")
 @AutoService({Connector.class})
 @Description("Dumps permissions from the HDFS.")
-public class HdfsPermissionExtractionConnector extends AbstractConnector {
+public class HdfsPermissionExtractionConnector extends AbstractConnector
+    implements HdfsPermissionExtractionDumpFormat {
 
   public HdfsPermissionExtractionConnector() {
-    super("hdfs-permissions");
+    super(FORMAT_NAME);
```

Comment:
The value passed to the `AbstractConnector` constructor is the connector name, not format name. I would suggest keeping `hdfs-permissions` here.

The `FORMAT_NAME` constant is used in a different way - see, for example: https://github.com/google/dwh-migration-tools/blob/fe1e3c5753055e64eb7df562d23692d9b514cf30/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java#L203-L204